### DISCUSSION TITLE
Have "HPE_INVALID_HEADER_TOKEN" issues when doing testing on CentOS VM

### DISCUSSION THREAD

**Discussion by ld0906** [artilleryio/artillery#1246] [Artillery]
Hello there,

Recently I met below error when doing testings on CentOS VM, This error cannot be reproduced 100%, but can reproduce 50%+ .
I am not sure if others also have this error.
Other informations:
 1: CentOS 7 (linux /x64)
 2. Node version: 16.13.0
 3.  artillery version: 1.7.9
 
Thanks!

Summary report @ 07:32:18(+0000) 2021-11-30
  Scenarios launched:  400
  Scenarios completed: 0
  Requests completed:  0
  Mean response/sec: 6.52
  Response time (msec):
    min: NaN
    max: NaN
    median: NaN
    p95: NaN
    p99: NaN
  Scenario counts:
    xxxxxxxxxx: 400 (100%)
  Errors:
    **HPE_INVALID_HEADER_TOKEN**: 400

**Comment by hassy** [comment]
hi @ld0906 - the error originates in Node.js itself because it's rejecting an invalid HTTP response header from the server. More information here: https://github.com/nodejs/node/issues/27711

It's quite odd that it's not 100% reproducible. Do you only see it under load? What's the stack of the system under test? (server, web framework etc)

**Comment by ld0906** [comment]
@hassy, Thanks for your updates. Yes, I see it under load. I am using it to test RESTful API (HTTP). It cannot reproduce 100% on same load.
